Resources/Private/Fusion/Root.fusion

```diff
@@ -9,4 +9,5 @@ prototype(Networkteam.Neos.Vite:Asset) {
     manifest = '.vite/manifest.json'
     outputPath = null
     entry = null
+    bareResource = false
```

Comment:
Maybe we should create another `AssetUri` prototype / implementation for this to differentiate it more (one generates HTML, the other is a URL - or URI in Flow-speak)?

Also in the `AssetIncludesBuilder` I would suggest separate methods to resolve just the entry URL since we don't need to consider recursion or CSS for this.

Reply:
Thanks for taking a look. I did a refactoring according to your recommendations. 
I decided to go with `AssetUrl` instead `-> AssetUri` beacause the returned value actually is a URL.